package.json

```diff
@@ -15,31 +15,26 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "peerDependencies": {
-    "apollo-cache-inmemory": "^1.1.5",
-    "apollo-client": "^2.2.0",
-    "apollo-link": "^1.0.7",
-    "apollo-link-http": "^1.3.2",
-    "bs-platform": "^2.1.0",
-    "graphql-tag": "^2.6.1",
-    "graphql_ppx": "^0.2.0",
-    "react-apollo": "^2.1.3"
+    "apollo-cache-inmemory": "^1.2.1",
+    "apollo-client": "^2.3.1",
+    "apollo-link": "^1.2.2",
+    "apollo-link-http": "^1.5.4",
+    "bs-platform": "^3.0.0",
+    "graphql-tag": "^2.9.2",
+    "graphql_ppx": "^0.2.1",
+    "react-apollo": "^2.1.4"
   },
-  "keywords": [
-    "Reason",
-    "Apollo",
-    "React",
-    "GraphQL"
-  ],
+  "keywords": ["Reason", "Apollo", "React", "GraphQL"],
   "author": "Grégoire Van der Auwermeulen <gregoirevandera@gmail.com>",
   "devDependencies": {
-    "apollo-cache-inmemory": "^1.1.5",
-    "apollo-client": "^2.2.0",
-    "apollo-link": "^1.0.7",
-    "apollo-link-http": "^1.3.2",
-    "bs-platform": "^2.1.0",
-    "graphql-tag": "^2.6.1",
-    "graphql_ppx": "^0.2.0",
-    "react-apollo": "^2.1.3",
-    "reason-react": "^0.3.1"
+    "apollo-cache-inmemory": "^1.2.1",
```

Comment:
We shouldn't force the users of the library to have the latest version right?

Reply:
I had a bug yesterday with a previous version of apollo-client, so let's keep it like that ;)